recurly_test.go

```diff
@@ -201,4 +201,9 @@ func TestPointerHelpers(test *testing.T) {
 	zPtr := Time(z)
 	// verify that recurly.Time(s) returns a pointer to s
 	t.Assert(*zPtr, z, "recurly.Time()")
+
+	strSlice := []string{"apple", "orange", "banana"}
+	slicePtr := StringSlice(strSlice)
```

Comment:
Trying to recall, did we end up not needing this helper since it's a pointer by default?

Reply:
I remember us talking about it, but I don't think the conversation was conclusive. I wrote a test anyway because I wasn't sure if we had decided to toss it. [This article](https://medium.com/swlh/golang-tips-why-pointers-to-slices-are-useful-and-how-ignoring-them-can-lead-to-tricky-bugs-cac90f72e77b) makes an argument for pointers to slices. If it matters, we aren't actually using `recurly.StringSlice()` anywhere in the tests or in the sample scripts.